holoviews/plotting/bokeh/element.py

```diff
@@ -942,16 +951,33 @@ class LegendPlot(ElementPlot):
         options. The predefined options may be customized in the
         legend_specs class attribute.""")
 
-
     legend_cols = param.Integer(default=False, doc="""
        Whether to lay out the legend as columns.""")
 
-
     legend_specs = {'right': dict(pos='right', loc=(5, -40)),
                     'left': dict(pos='left', loc=(0, -40)),
                     'top': dict(pos='above', loc=(120, 5)),
                     'bottom': dict(pos='below', loc=(60, 0))}
 
+    def _process_legend(self, plot=None):
```

Comment:
I see this was moved from ``PointPlot`` and I expect it is more general here. That said, does this have anything to do with colormapping or is it just an unrelated improvement?